src/common/definitions.js

```diff
@@ -963,6 +963,10 @@ const teraQuestions = {
   toxicExposureEndDate: date,
 };
 
+const yesNoSchema = {
```

Comment:
I think this is good - it's clearer. I think we should add a comment that says that it should be kept in line with the yesNoSchema in https://github.com/department-of-veterans-affairs/vets-website/blob/39e38ef10d72d94b33d18e699a1495b1ae8bf68d/src/platform/forms-system/src/js/web-component-patterns/yesNoPattern.jsx#L70